src/SignaturePad.UWP/SignaturePad.xaml.cs

```diff
@@ -0,0 +1,338 @@
+                this.InkCanvas.SizeChanged -= InkCanvas_SizeChanged;
+        }
+
+
+        private void OnIsBlankChanged(bool isblank)
+        {
+            IsBlankChanged?.Invoke(this, isblank);
+        }
+
+        #region save image
+
+        public async Task<Stream> GetImageStreamAsync(CanvasBitmapFileFormat type)
+        {
+            // do not put in using{} structure - it will close the shared device.
+            var device = CanvasDevice.GetSharedDevice();
+            var width = (float)this.InkCanvas.ActualWidth;
+            var height = (float)this.InkCanvas.ActualHeight;
+            var currentDpi = DisplayInformation.GetForCurrentView().LogicalDpi;
+
+            using (var fileStream = new InMemoryRandomAccessStream())
+            {
+                using (var offscreen = new CanvasRenderTarget(device, width, height, currentDpi))
+                {
+                    using (var session = offscreen.CreateDrawingSession())
+                    {
+                        session.Clear(Colors.Transparent);
+                        session.DrawInk(this.InkCanvas.InkPresenter.StrokeContainer.GetStrokes());
+                    }
+
+                    await offscreen.SaveAsync(fileStream, type);
+
+                    var stream = fileStream.AsStream();
+                    stream.Position = 0;
+                    return stream;
+                }
+            }
+        }
+
+        public async Task<string> ConvertToBase64Async(CanvasBitmapFileFormat type)
+        {
+            var stream = await GetImageStreamAsync(type);
+            var bytes = StreamToBytes(stream);
+            var base64String = BytesToBase64String(bytes);
+
+            return base64String;
+        }
+
+        private byte[] StreamToBytes(Stream input)
+        {
+            var memStream = StreamToMemoryStream(input);
+            var bytes = memStream.ToArray();
+            memStream.Dispose();
+
+            return bytes;
+        }
+
+        /// <summary>
+        /// Need to dispose stream after using
+        /// </summary>
+        /// <param name="input"></param>
+        /// <returns></returns>
+        private MemoryStream StreamToMemoryStream(Stream input)
+        {
+            byte[] buffer = new byte[16 * 1024];
+            var ms = new MemoryStream();
+            int read;
+
+            while ((read = input.Read(buffer, 0, buffer.Length)) > 0)
+            {
+                ms.Write(buffer, 0, read);
+            }
+
+            return ms;
+        }
+
+        private string BytesToBase64String(byte[] bytes)
+        {
+            return Convert.ToBase64String(bytes);
+        }
+
+        #endregion
+
+        public void LoadPoints(Point[] loadedPoints)
+        {
+            if (loadedPoints == null || loadedPoints.Count() == 0)
+                return;
+
+            var wasBlank = IsBlank;
+
+            // unfortunately UWP does not allow to create InkStrokes from code..... crap...
```

Comment:
Your statement is incorrect. You can create an `InkStroke` on UWP with `InkStrokeBuilder`.

    var strokeBuilder = new InkStrokeBuilder();
    var stroke = strokeBuilder.CreateStrokeFromInkPoints(loadedPoints.Select(p => new InkPoint(p, 1f)), Matrix3x2.Identity);
    InkCanvas.InkPresenter.StrokeContainer.AddStroke(stroke);